### PR TITLE
JAMES-3565 Use apache/james images

### PR DESCRIPTION
**Pull request by chibenwa**
https://issues.apache.org/jira/browse/JAMES-3565

https://www.mail-archive.com/server-dev@james.apache.org/msg70107.html

### CODE REVIEW

File: dockerfiles/run/docker-compose.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ services:
       - rabbitmq
       - swift
```
Comment (Arsnael) [inline dockerfiles/run/docker-compose.yml:11]: Did you try to run this locally? I doubt it works with this old swift object storage version

File: dockerfiles/run/docker-compose.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ services:
...
     entrypoint: bash -c "java -Dworking.directory=/root/ -Dlogback.configurationFile=/root/conf/logback.xml -jar james-server.jar"
-    image: linagora/james-rabbitmq-project:latest
+    image: apache/james:distributed-3.6.0
```
Comment (jeantil) [inline dockerfiles/run/docker-compose.yml:13]: is there a doc on what the naming scheme is  ? (I created a custom SMTP only server, I wasn't too sure how to name it) if the naming scheme is clear I intend to align with it before proposing this assembly back to the project.

**Review by jeantil — COMMENTED** [review]
In addition to my inline comment about the naming scheme, could we build a capability matrix of the various assemblies/images ? 

something like

| name  | deployment model | general persistence | blob persistence | queue type |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| distributed  | distributed  |  cassandra | S3 blobstore | rabbitMQ
| demo  | single node  | jpa (mysql) | filesystem | in memory

etc.

even though I start to be familiar with the project, I regularly get lost in the various assemblies :)

**Comment by chibenwa** [comment]
http://james.apache.org/server/packaging.html sounds like the right page

**Comment by jeantil** [comment]
> http://james.apache.org/server/packaging.html sounds like the right page

awesome, I'll try and find some time to move that to the antora docs and make sure it's aligned with the actual packages (that page doesn't mention rabbit anywhere and I'm pretty sure we have at least 2 assemblies using rabbit.

**Comment by chibenwa** [comment]
One assembly. We got rid of the ldap variation, bundled in the base package.

I'll see for refreshing this page, but in another pr...

**Comment by chibenwa** [comment]
If there's comments I agree fixing them in another PR. I'll move on.

**Review by Arsnael — APPROVED** [review]

File: docs/modules/servers/pages/15-minute-demo.adoc [hunk 3]
```diff
@@ -87,7 +71,7 @@ List all the current users:
 
 [source,bash]
 ----
-./james listUsers
+docker exec james james-clivv listUsers
```
Comment (rouazana) [inline docs/modules/servers/pages/15-minute-demo.adoc:74]: clivv?

File: docs/modules/servers/pages/15-minute-demo.adoc [hunk 4]
```diff
@@ -96,28 +80,28 @@ Create a new "test.local" domain:
...
-./james addDomain test.local
+docker exec james james-cli addDomain test.local
 ----
 
 List the domains again to ensure that "test.local" has successfully been added:
 
 [source,bash]
 ----
-./james listDomains
+docker exec james james-clivv listDomains
```
Comment (rouazana) [inline docs/modules/servers/pages/15-minute-demo.adoc:90]: idem

File: docs/modules/servers/pages/15-minute-demo.adoc [hunk 5]
```diff
@@ -96,28 +80,28 @@ Create a new "test.local" domain:
...
 ----
 
 Add the user "testuser" to the "test.local" domain with password "password":
 
 [source,bash]
 ----
-./james addUser testuser@test.local password
+docker exec james james-cliv addUser testuser@test.local password
```
Comment (rouazana) [inline docs/modules/servers/pages/15-minute-demo.adoc:97]: cliv here

Comment (chibenwa) [reply docs/modules/servers/pages/15-minute-demo.adoc:97]: I have a sensibility issue on my keyboard, it is very annoying...

**Comment by chibenwa** [comment]
See https://github.com/apache/james-project/pull/398

**Pull request merged**